### PR TITLE
Team practices

### PR DESCRIPTION
**Pull request by adborden**
Hey folks, I'm assigning all the team members to explicitly Approve :white_check_mark: these team practices. Feel free to make suggestions, edits, or add any practices that haven't been captured. I borrowed these from the FOIA project and adapted them based on what I've seen (and some of my own opinions :smirk_cat:).

I've also included the boilerplate 18F LICENSE, CONTRIBUTING, and README content.

### CODE REVIEW

**Review by eddietejeda — APPROVED** [review]

**Review by onezerojeremy — APPROVED** [review]
looks good to me!

**Review by hbillings — APPROVED** [review]
I just capitalized "Slack" and "Performance.gov" in the GH editor -- didn't realize it would push directly to your branch, sorry! Otherwise, looks good!

File: team-practices.md
Comment (vicuk) [inline team-practices.md]: The definition of what is in the backlog doesn't match the exit criteria. It seems more that you my want to say "Items ready to be groomed and prioritized in preparation to be pulled into the In Progress column . . . "

File: team-practices.md
Comment (vicuk) [inline team-practices.md:63]: If grooming is part of the exit criteria, we may want to remove it from the definition of what is in the backlog above.

File: team-practices.md
Comment (vicuk) [inline team-practices.md]: Same thing here. If the Exit criteria for In Progress is a reviewed, approved, and merged PR, then the definition of what is Complete should change to match it. It currently says open PR, but it can't be opened if it had to be merged to leave In Progress. Unless I'm misunderstanding.

File: team-practices.md
Comment (vicuk) [inline team-practices.md:88]: This is also an exit criteria for In Progress. I might be misunderstanding.

File: team-practices.md
Comment (vicuk) [inline team-practices.md]: Same thing. If the Exit criteria for In Progress is a reviewed, approved, and merged PR, then the definition of what is Complete should change to match it. It currently says open PR, but it can't be opened if it had to be merged to leave In Progress. Unless I'm misunderstanding.

**Review by vicuk — CHANGES_REQUESTED** [review]
I left some comments.

File: team-practices.md
Comment (adborden) [reply team-practices.md]: You are correct. Which way do you think it should work? I'm leaning toward  the In Progress exit criteria would be that the bulk of the work is done, so it is ready for feedback or validation through usability testing. That would mean that we remove this exit criteria.

File: team-practices.md
Comment (adborden) [reply team-practices.md:88]: Good point, we should re-word this so it is more clear. My intention is that this column is about being peer-reviewed, so while the acceptance criteria was completed in "In Progress", a team member should review it so they concur that the acceptance criteria has indeed been completed. Does that make sense?

Maybe we should rename the column from "Complete" to "Review" or "Validate"?

Comment (vicuk) [reply team-practices.md:88]: Yeah, I think that would clarify it. I think we usually say "review" in our process boards, but "validate" works as well. I'm indifferent. I'll trust your wording. And I think this works with removing the reviewed, approved, and merged PRs from the In Progress exit criteria. Thanks!!

File: team-practices.md
Comment (vicuk) [reply team-practices.md]: I like your leaning. That also works with what you said below about creating a "Validate" or "Review" column for peer review where PRs would be reviewed, approved, and merged.

File: team-practices.md
Comment (adborden) [reply team-practices.md:88]: 👍 I'm going to go with "Review/Validate" for the column name.

Comment (vicuk) [reply team-practices.md:88]: 👍

**Comment by adborden** [comment]
@vicuk I just pushed an update, let me know if that works for you.

**Comment by adborden** [comment]
@vicuk if you're set, do you mind changing your review from "changes requested" to "approved"?

**Comment by vicuk** [comment]
@adborden do you want me to change the category/exit criteria for the icebox and backlog. I think it was the same issue, so just changing the backlog to say "ready for grooming" rather than "groomed" might work. And then we keep "groomed" in the exit criteria for the backlog. How does that sound? If good, are you changing that or are you asking me to?

**Review by vicuk — APPROVED** [review]

**Comment by adborden** [comment]
@tram I'm going to merge this, but I still want your :white_check_mark: when you get back. When you have a chance, look it over and let me know if you have any changes/additions.

**Comment by eddietejeda** [comment]
@tram any updates on ✅ this issue.

**Comment by tram** [comment]
Gave the 👍 to @adborden's comment (I especially like the mid-week retro), but am not sure how to ✅ it after being merged. ¯\_(ツ)_/¯

**Pull request merged**
